helm/prometheus-rules/templates/alerting-rules/kyverno.all.rules.yml

```diff
@@ -0,0 +1,76 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: kyverno.all.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: webhooks
+    rules:
+    - alert: KyvernoWebhooksPresent
+      annotations:
+        description: 'Kyverno has no available replicas but webhooks are present.'
+        opsrecipe: kyverno-webhooks/
+      expr: sum(kube_validatingwebhookconfiguration_info{validatingwebhookconfiguration=~"kyverno-.*"}) > 0 and sum(kube_deployment_status_replicas{deployment=~"kyverno|kyverno-admission-controller"}) == 0
+      for: 5m
+      labels:
+        area: managedservices
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_status_updating: "true"
+        cancel_if_outside_working_hours: "false"
+        severity: page
+        team: shield
+        topic: kyverno
+  - name: reports
+    rules:
+    - alert: KyvernoReportsTooHigh
+      annotations:
+        description: '{{`Kyverno {{ $labels.kind }} are too high.`}}'
+        opsrecipe: kyverno-reports/
+      expr: aggregation:kyverno_resource_counts{kind=~"(clusteradmissionreports|admissionreports).kyverno.io"} > 50000
+      for: 15m
+      labels:
+        area: managedservices
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_status_updating: "true"
+        cancel_if_outside_working_hours: "false"
```

Comment:
I think it could be possible, the thing is if this happens during a weekend/holiday then we could avoid issues before they happen. We could also increase the threshold that would slow the alert from triggering.